sky/jobs/state.py

```diff
@@ -148,12 +148,13 @@ def create_table(cursor, conn):
                                  'TEXT DEFAULT NULL',
                                  value_to_replace_existing_entries='default')
 
-    db_utils.add_column_to_table(cursor,
-                                 conn,
-                                 'job_info',
-                                 'priority',
-                                 'INTEGER',
-                                 value_to_replace_existing_entries=500)
+    db_utils.add_column_to_table(
+        cursor,
+        conn,
+        'job_info',
+        'priority',
+        'INTEGER',
+        value_to_replace_existing_entries=constants.DEFAULT_PRIORITY)
```

Comment:
We need to add a migration for people that may already have the old default value in the column type.